backend/src/libs/packages/encrypt/base-encrypt.package.ts

```diff
@@ -0,0 +1,17 @@
+import bcrypt from 'bcryptjs';
+
+import { config } from '#libs/packages/config/config.js';
+
+import { type Encrypt } from './libs/types/types.js';
+
+class BaseEncrypt implements Encrypt {
+  public async generateSalt(rounds?: number): Promise<string> {
```

Comment:
```suggestion
  public async generateSalt(rounds: number): Promise<string> {
```
Make it required as it was [requested](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/46#discussion_r1303862151) by Liza